src/multidim/intervalbox.jl

```diff
@@ -138,6 +138,26 @@ as a vector.
     end
 end
 
+@generated function mince(x::IntervalBox{N,T}, tup::NTuple{N,Int}) where {N,T}
+    quote
+        n = maximum(tup)
+        nodes_matrix = Array{Interval{T},2}(undef, n, N)
+        fill!(nodes_matrix, emptyinterval(T))
```

Comment:
```suggestion
        nodes_matrix = fill(emptyinterval(T), n, N)
```
should be equivalent.

Reply:
Thanks! Done!